_maps/RandomRuins/SpaceRuins/onehalf.dmm

```diff
@@ -875,8 +876,8 @@
 "cI" = (
 /obj/structure/safe/floor,
 /obj/item/tank/internals/oxygen/red,
-/obj/item/clothing/mask/gas/syndicate,
-/obj/item/clothing/suit/space/hardsuit/syndi,
+/obj/item/clothing/head/helmet/space/eva,
```

Comment:
Given this is in a ruin exposed to space, you may as well ditch the eva gear and replace it with some cash money.